### PR TITLE
EFA-625 Restore planck dbs

### PR DESCRIPTION
**Pull request by ig-garcia**

### CODE REVIEW

File: k9mail/src/main/java/com/fsck/k9/planck/ui/activities/provisioning/ProvisioningActivity.kt [hunk 1]
```diff
@@ -1,53 +1,119 @@
...
+        binding = ActivityProvisioningBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+        waitingForProvisioningText = binding.waitingForProvisioningText
+        progressBar = binding.provisioningProgress
+        binding.provisioningSkipButton.setOnClickListener { this.viewModel.initializeApp() }
+        binding.provisioningRestoreDataButton.setOnClickListener { pickFolder() }
+
+        folderPickerLauncher =
+            registerForActivityResult(ActivityResultContracts.StartActivityForResult()) { result ->
+                Log.e("EFA-625", "RESULT CODE: ${result.resultCode}, DATA: ${result.data}")
```
Comment (Stereodark) [inline k9mail/src/main/java/com/fsck/k9/planck/ui/activities/provisioning/ProvisioningActivity.kt:102]: so we want to reference the ticket ID in the log entry? What's the upside?

Comment (Stereodark) [reply k9mail/src/main/java/com/fsck/k9/planck/ui/activities/provisioning/ProvisioningActivity.kt:102]: it's just a debug line I guess

**Review by Stereodark — COMMENTED** [review]
restore looks fine, as discussed in the ticket with MDM out of scope and not using and standard Samsung/Google functionality as for those you need to be in the store, or the app backup won't work. However, I don't get how do we export the folders, I might have missed the functionality, but how does the backup get produced?

File: k9mail/src/main/java/com/fsck/k9/planck/ui/activities/provisioning/ProvisioningActivity.kt (hunk 1 above)
Comment (ig-garcia) [reply k9mail/src/main/java/com/fsck/k9/planck/ui/activities/provisioning/ProvisioningActivity.kt]: was the moment of change.

Comment (ig-garcia) [reply k9mail/src/main/java/com/fsck/k9/planck/ui/activities/provisioning/ProvisioningActivity.kt]: removed it thanks.

**Pull request merged**